static/js/app.js

```diff
@@ -26,3 +26,12 @@ function timeDifference(current, previous) {
     return 'approximately ' + Math.round(elapsed / msPerYear) + ' years ago'
   }
 }
+
+// select all leaderboards
+var allLeaders = document.querySelectorAll("div.org ul");
+  allLeaders.forEach(function(orgList) {
+  // shuffle each leaderboard
+  for(var i = orgList.children.length; i >= 0; i--) {
+    orgList.appendChild(orgList.children[Math.random()*i | 0]);
+  }
+});
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `all.whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpd5h9drm4/static/js/app.js
+++ b/tmp/tmpd5h9drm4/static/js/app.js
@@ -34,4 +34,4 @@
   for(var i = orgList.children.length; i >= 0; i--) {
     orgList.appendChild(orgList.children[Math.random()*i | 0]);
   }
-});+});
```